### PR TITLE
Move SaveTokensAsClaims to base RemoteAuthOptions

### PR DESCRIPTION
**Pull request by HaoK**
cc @Tratcher

### CODE REVIEW

File: src/Microsoft.AspNet.Authentication/RemoteAuthenticationOptions.cs [hunk 1]
```diff
@@ -48,6 +48,15 @@ public string DisplayName
...
 
+        /// <summary>
+        /// Defines whether access and refresh tokens should be stored in the
+        /// <see cref="ClaimsPrincipal"/> after a successful authentication.
+        /// This property is set to <c>false</c> by default to reduce
+        /// the size of the final authentication cookie.
+        /// </summary>
+        public bool SaveTokensAsClaims { get; set; }
+
+
```
Comment (PinpointTownes) [inline src/Microsoft.AspNet.Authentication/RemoteAuthenticationOptions.cs]: Too many line breaks :trollface:

**Comment by Eilon** [comment]
:shipit: after @PinpointTownes 's whitespace :smile:

**Comment by PinpointTownes** [comment]
Don't forget to remove the assignations in the OAuth2 providers :clap: 

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Facebook/FacebookOptions.cs#L27
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Google/GoogleOptions.cs#L25
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.MicrosoftAccount/MicrosoftAccountOptions.cs#L25

**Comment by HaoK** [comment]
f24c35f1a93ed0d6de3cda2b4ab15906f5f6d56b

**Comment by PinpointTownes** [comment]
Now, I have a bunch of providers to update... :smile:

**Comment by HaoK** [comment]
Deleting lines is always good right? :)

**Comment by PinpointTownes** [comment]
Yeah, specially when it makes things consistent :clap:

**Pull request closed without merging**